src/main/java/backend/RepoId.java

```diff
@@ -0,0 +1,69 @@
+package backend;
+
+import org.eclipse.egit.github.core.RepositoryId;
+import util.Utility;
+
+/**
+ * Represents the ID of a repository. To be used to ensure safe comparison
+ */
+public final class RepoId {
+
+    //repoOwner can be an org ID or username
+    private final String repoOwner;
+    private final String repoName;
+
+    public RepoId(String repoIdString) throws IllegalArgumentException {
+        throwExceptionIfNotWellFormed(repoIdString);
```

Comment:
Don't think we need the `throws` declaration here, since it's a runtime exception.

Also, in general we can leave out the types of variables from their names, since that's meta-information, and it's usually apparent from the declaration anyway. :stuck_out_tongue: